src/containers/app-shell.js

```diff
@@ -346,6 +347,7 @@ class App extends PureComponent {
         {header}
         {boxJSX}
         {this.props.children}
+        <Footer />
```

Comment:
If our about-us page is wrapped in `app-shell`, the [`footer`](https://github.com/nickhsine/twreporter-react/blob/1d05a1f61a65245447dd8bd9f8b2a95dd5e1ff32/src/components/about-us/index.js#L90) in the page itself should be removed.